tools/generators/falso/index.ts

```diff
@@ -2,21 +2,40 @@ import { Tree, getProjects, formatFiles, names } from '@nrwl/devkit';
 import { join } from 'path';
 import { appendFile } from 'fs/promises';
 
-export default async function (tree: Tree, { name, project }: { name: string, project: string }) {
-
+export default async function (
+  tree: Tree,
+  { name, project, json }: { name: string; project: string; json: boolean }
+) {
   const sourceRoot = getProjects(tree).get(project)?.sourceRoot;
 
   const n = names(name);
 
   if (sourceRoot) {
-    tree.write(join(sourceRoot, 'lib', `${n.fileName}.ts`), ` 
-      import { rand } from './core';
-      
-      export function ${n.propertyName}() {
-        return rand([
-        ])
+    if (json) {
+      tree.write(
+        join(sourceRoot, 'lib', `${n.fileName}.json`),
+        `
+        {
+          "data": [
+          ]
+        }
+      `
+      );
+    }
+
+    tree.write(
+      join(sourceRoot, 'lib', `${n.fileName}.ts`),
+      `
+      import { FakeOptions, fake } from './core';
+      ${json ? `import { data } from './${n.fileName}.json'` : ''}
+
+      export function ${
+        n.propertyName
+      }<Options extends FakeOptions>(options?: Options) {
+        return fake(${json ? 'data' : '[]'}, options);
       }
```

Comment:
Please add also a spec file.

Reply:
Meaning to also generate the `.spec` file for the to generate falso right?
e.g. `yarn new --name dummy --json` produces `.../tests/dummy.spec.ts` right?

Do we want a explicit option to include the spec file (`--test`) ?

Reply:
Falso generator will create a matching spec file by default.
Added `--skip-test` to change default behaviour.